public/locales/ru/common.json

```
@@ -0,0 +1,15 @@
+{
+  "title": "Самый простой способ создать",
+  "readme": "README",
+  "description": "Наш простой редактор позволяет быстро добавлять и настраивать все необходимые разделы для readme вашего проекта.",
+  "get-started": "Начать",
+  "100-free": "100% бесплатный",
+  "made-with-love": "Сделано с ",
+  "by": " от",
+  "language": "Язык",
+  "english": "Английский",
+  "french": "Французский",
+  "spanish": "Испанский",
+  "russian": "Русский",
```

@Awilum you seemed to have missed Italian over here. Could please add that

---

but it is added here in the merge: https://github.com/katherinepeterson/readme.so/pull/76/commits/1a66d08102add9d2708d2088498c434343e49435

---

I'm sorry @Awilum, I don't think I communicated the issue correctly. I was just trying to point out at the Russian translation for `Italian` was missing in your `/ru/common.json` file. 

So if you check the preview and choose Russian, the language drop down show `Italian` in English and not in Russian. Ideally this should fix the issue.

```suggestion
  "italian": "Итальянский",
  "russian": "Русский",
```

---

I see.
done https://github.com/katherinepeterson/readme.so/pull/76/commits/92fe93f2f354b40036159eb18497f806e2852f76